browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -1077,7 +1099,7 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 			} else {
 				// Row rectangle
 				topLeftTwips = new L.Point(parseInt(0), parseInt(strTwips[1]));
-				offset = new L.Point(parseInt(maxRow), parseInt(strTwips[4]));
+				offset = new L.Point(parseInt(maxRow), parseInt(strTwips[3]));
```

Comment:
this changes behavior of: `!textMsg.startsWith('updateHighlight')` case

is that valid?

Reply:
Not a mistake. Planned. It was wrong before, i fixed it in that commit.